### ISSUE TITLE
Not showing conflicts on files defined as optional

### ISSUE THREAD

**Issue by Shine753** [ModOrganizer2/modorganizer#1105]
When 2 mods have the same optional files (in an "optional" folder), they are not used by the mod but still shown as in conflict.

**Comment by Al12rs** [comment]
I'm not sure I want to remove this, for me conflicts are there to indicate which files are contained in both mods. The only file that is currently being excluded is meta.ini and that is understandable since all mods would be conflicting otherwise.
I would still find it valuable to know that two mods contain the same optional files. You might want to delete one since you already have the copy somewhere else for example or something like that.

**Comment by Shine753** [comment]
I think I understand your point of view.
Do you mean that you make a difference between "optional" and hidden ?
If I don't want a possible conflict to be found, I have to hide the "optional" file, or delete it. Is that right ?

**Comment by isanae** [comment]
MO shows conflicts between files and does not distinguish between hidden, optional or regular files, nor would I want it to. The features for hiding files or marking esps as optional are purely for convenience.

These files are not treated differently by usvfs and so are part of the virtual filesystem that the program interacts with. If you run explorer++ from MO, you'll see your hidden or optional files. If these files are shown to programs, then MO must include them in conflict detection.

**Comment by Al12rs** [comment]
Closing since I'm keeping this open in case we implement VFS file hiding instead of mohidden, which would allow us to simply exclude the optional folder from the vfs and the conflicts as well.

**Comment by StormFX** [comment]
I actually stopped by to request an option to disable the display of hidden files in the _Conflicts_ tab. Hopefully, the VFS hiding idea works out. It pokes my OCD to see the overwrites/overwritten flag(s) in the mod list. :p

**Comment by StormFX** [comment]
Just throwing this out there, but how about having MO read exclusively from a `Data` subdirectory? This would allow files and folders to be stored within the mod's directory, but out of the scope of the VFS. Besides, half the archives out there already include a `Data` folder. :p (An honest, if slightly sarcastic inquiry :p )

**Comment by Al12rs** [comment]
No, we can't just reinvent how Mo2 works, we have existing setups of our users. Moving files is also exactly what Mo2 tries to avoid by using a VFS in the first place.
